packages/message-manager/src/utils.ts

```diff
@@ -93,7 +94,7 @@ export function validateTypedSignMessageDataV1(
  */
 export function validateTypedSignMessageDataV3V4(
   messageData: TypedMessageParams,
-  currentChainId: string | undefined,
```

Comment:
Can this still be `undefined`? I see this in `TypedMessageManager.addUnapprovedMessageAsync`: 

``` typescript
      const currentChainId = this.getCurrentChainId?.();
      validateTypedSignMessageDataV3V4(messageParams, currentChainId);
```

Reply:
Yep! It's declared here as optional though.

Reply:
They're the same now, but I'm wondering whether they will be the same when we enable `exactOptionalPropertyTypes` (assuming we want to revive [this PR](https://github.com/MetaMask/core/pull/796)).

Reply:
Updated in https://github.com/MetaMask/core/pull/1367/commits/4407595c53a3fd744c7d31142f6eb5019e82503d